Kernel_23/include/CGAL/determinant.h

```diff
@@ -39,7 +39,7 @@ determinant(
  const RT& a10,  const RT& a11)
 {
 // First compute the det2x2
-  const RT m01 = a00*a11 - a10*a01;
+  const RT m01 = (a00*a11 - a10*a01);
```

Comment:
Strange. What was the warning, that an extra parenthesis can fix?

Reply:
Can't find it again :/ Something about a wrong conversion between FT and RT but I don't think it fixed it.